torch/fx/experimental/validator.py

```diff
@@ -363,6 +363,7 @@ def __getattr__(self, name: str) -> Any:
                 "not_": z3.Not,
                 "floor": self._ops.floor,
                 "ceil": self._ops.ceil,
+                "minimum": self._ops.min,
```

Comment:
@andrewlee302 Could you briefly explain why do we need to change this? Thanks!

Reply:
Update the commit, to address torchbench error on some models. Log failure snippet ([raw log](https://ossci-raw-job-status.s3.amazonaws.com/log/18915848660)):
```
2023-11-22T04:23:29.6156040Z   File "/opt/conda/envs/py_3.10/lib/python3.10/site-packages/torch/fx/experimental/validator.py", line 372, in __getattr__
2023-11-22T04:23:29.6156964Z     raise AttributeError(f"unhandled operator: {name}")
2023-11-22T04:23:29.6157656Z torch._dynamo.exc.InternalTorchDynamoError: unhandled operator: minimum
```

Local test passes. A command to repro is:
```
python benchmarks/dynamo/torchbench.py --ci --accuracy --timing --explain --inductor --device cpu --inference --float32 --total-partitions 1 --partition-id 0 --only vision_maskrcnn
```

Reply:
I'd guess it somehow being related to adding `sympy.Min` to the IR in this PR. @ezyang does this change look good to you? I believe, this just adds a key to the pre-existing `REPLACEMENT` dict which isn't likely to disrupt anything?

Reply:
yeah, this should be fine. Maybe do maximum too?